job_spec/WATER_MAP.yml

```diff
@@ -153,6 +156,9 @@ WATER_MAP:
       timeout: 36000
       vcpu: 1
       memory: 126000
+      secrets:
+        - EARTHDATA_USERNAME
+        - EARTHDATA_PASSWORD
```

Comment:
:thinking: I'm not actually sure if `asf_tools` needs EDL credentials... will need to check.

Reply:
It does not.